### PR TITLE
chore: Change time format

### PR DESCRIPTION
**Pull request by arian81**
## Description

Please include a concise summary, in clear English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #7179

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed.

### CODE REVIEW

**Comment by abidlabs** [comment]
Thanks @arian81! You addressed my suggestion from #7270 which was to extract the logic into a separate helper function, so this looks good.

As mentioned in #7270, there is actually one more instance of time formatting that needs to be fixed, which is here:

https://github.com/gradio-app/gradio/blob/21cfb0acc309bb1a392f4d8a8e42f6be864c5978/js/video/shared/Player.svelte#L80

**Comment by arian81** [comment]
> Thanks @arian81! You addressed my suggestion from #7270 which was to extract the logic into a separate helper function, so this looks good.
> 
> As mentioned in #7270, there is actually one more instance of time formatting that needs to be fixed, which is here:
> 
> https://github.com/gradio-app/gradio/blob/21cfb0acc309bb1a392f4d8a8e42f6be864c5978/js/video/shared/Player.svelte#L80

I will change that instance. I had one question tho and that is why the formatting logic for video player different than the rest. Currently i didn't apply the same function to the video player because i'm not even sure which component is using that one. But it would nice if you could clarify this.

**Comment by abidlabs** [comment]
I believe its the static Video component that is using this. So if you were to do:

```py
import gradio as gr

with gr.Blocks() as demo:
    gr.Video("test.mp4")
    
demo.launch()
```

you'll see that even with your changes, its still using the mm:ss format right now

**Comment by arian81** [comment]
> I believe its the static Video component that is using this. So if you were to do:
> 
> ```python
> import gradio as gr
> 
> with gr.Blocks() as demo:
>     gr.Video("test.mp4")
>     
> demo.launch()
> ```
> 
> you'll see that even with your changes, its still using the mm:ss format right now

ok i changed that one to use the global function as well. Everything seems to be working now.

File: js/utils/src/utils.ts [hunk 1]
```diff
@@ -151,6 +151,21 @@ async function copy_to_clipboard(value: string): Promise<boolean> {
...
+	const paddedSeconds = `${
+		secondsRemainder < 10 ? "0" : ""
+	}${secondsRemainder}`;
+
+	if (hours > 0) {
+		return `${hours}:${paddedMinutes}:${paddedSeconds}`;
+	}
+	return `${minutes}:${paddedSeconds}`;
+};
+
```
[the remark this answers is not in the document]
Comment (arian81) [reply js/utils/src/utils.ts]: done

**Comment by arian81** [comment]
@abidlabs Something seems to be broken in the upload button causing the functional test to fail but I haven't changed that file.

**Comment by abidlabs** [comment]
I think that's just a flaky functional test we need to get under control, don't worry about it!

**Review by abidlabs — APPROVED** [review]
LGTM thanks @arian81!

**Comment by jjshoots** [comment]
Thanks for this everyone! Needed this to get something blocking on my end moving. :)

**Pull request merged**